run.cmd

```diff
@@ -0,0 +1,16 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+if not defined VisualStudioVersion (
+  if defined VS140COMNTOOLS (
+    call "%VS140COMNTOOLS%\VsDevCmd.bat"
+    goto :Run
+  )
+  echo Error: Visual Studio 2015 required.
```

Comment:
What is the plan for moving this to VS 2017?  The dotnet/sdk repo already depends on VS 2017 being installed.